rasa/nlu/featurizers/dense_featurizer/spacy_featurizer.py

```diff
@@ -20,6 +21,9 @@
     from spacy.tokens import Doc
 
 
+logger = logging.getLogger(__name__)
```

Comment:
You probably added this feature because you were debugging. But it may need to be removed now? Just checking.

Reply:
It is used further down in the file (line 73).
```
            logger.debug("No features present. You are using an empty spaCy model.")
```

Reply:
d0h! I forogot.